frontend/packages/schema-editor/src/components/SchemaTree/SchemaNode/ActionButtons/AddPropertyMenu.tsx

```diff
@@ -68,7 +68,7 @@ type PropertyItems = {
   icon: (IconProps: IconProps) => JSX.Element;
 };
 
-const propertyItems: PropertyItems[] = [
+export const propertyItems: PropertyItems[] = [
```

Comment:
Denne er ikke eksportert av samme grunn som jeg nevnte i en annen kommentar. Vi kan godt bruke denne i andre komponenter, men da må vi flytte den ut av komponentmappen.

Jeg ser for øvrig at bruken av den er ganske lik begge steder. Har du vurdert å lage en felles komponent for begge stedene, hvor hele menyen er innebygget, i stedet for å bare gjenbruke `propertyItems`-listen?

Reply:
den også har noe til å spørre deg om på mandag